CMakeLists.txt

```diff
@@ -70,6 +72,16 @@ endif()
 
 add_definitions("-DEIGEN_QLD_EXPORT")
 
+# For MSVC, set local environment variable to enable finding the built dll
+# of the main library when launching ctest with RUN_TESTS
+if(MSVC)
+  if(USE_FORTRAN_SUBDIRECTORY_DEFAULT)
```

Comment:
Shouldn't this be `USE_FORTRAN_SUBDIRECTORY`?

Reply:
Shouldn't change anything in most case, but you're right. Fixed.